demo.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang = "en">
+<head>
+  <meta charset="UTF-8">
+  <title>Build Your Own Bootstrap</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  <div>
+    <header>
+      <h1 class="section-title">Headings</h1>
+      <p>To style headings in various sizes, add the
+        <code class="class-name">.headings</code> class to your
+        <code class="element-in-text">&lt;h1&gt;</code> through
+        <code class="element-in-text">&lt;h6&gt;</code> tags.</p>
```

Comment:
I would drop this closing p tag on the next line so it keeps the page orientation in a good readable format.